sympy/solvers/ode/tests/test_ode.py

```diff
@@ -2211,3 +2211,11 @@ def test_issue_17322():
     sol = [Eq(f(x), C1), Eq(f(x), C1*exp(-x))]
     assert set(sol) == set(dsolve(eq, hint='lie_group'))
     assert checkodesol(eq, sol) == 2*[(True, 0)]
+
+
+def test_issue_4414():
```

Comment:
I think it is better to add this test near the closest test number, rather than at the end of the file

Reply:
The test should be added to the refactored tests in `test_single.py` along with the other cases for `2nd_linear_bessel`.

A link to the issue can be included in a comment.